pwnlib/commandline/libcdb.py

```diff
@@ -57,6 +57,20 @@
     help = 'Do NOT attempt to unstrip the libc binary with debug symbols from a debuginfod server'
 )
 
+lookup_parser.add_argument(
+    '--no-offline',
+    action = 'store_true',
+    default = False,
+    help = 'Disable offline libcdb search mode'
+)
+
+lookup_parser.add_argument(
+    '--no-online',
```

Comment:
```suggestion
    '--offline',
```
This should be as short and as intuitive as possible.

I even think disabling online search should be left for a separate PR.